include/u.h

```diff
@@ -14,14 +14,14 @@
 #endif
 
 #define nil		((void*)0)
-typedef	unsigned short	ushort;
+typedef	uint16_t	ushort;
 typedef	unsigned char	uchar;
-typedef unsigned long	ulong;
-typedef unsigned int	uint;
+typedef uint32_t	ulong;
+typedef uint32_t	uint;
 typedef   signed char	schar;
-typedef	long long	vlong;
-typedef	unsigned long long uvlong;
-typedef	unsigned int		Rune;
+typedef	int64_t	vlong;
+typedef	uint64_t uvlong;
```

Comment:
In fact, that u.h should be gone, but "out of the tree". If we use u.h we need harvey tree. It's used in ap/plan9 sources.